template/src/store.ts

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
Hmm.  I just realized we were leaving `store.ts/js` directly in `src`.  I'd rather put that in `src/app/store.ts` for both of them.

Reply:
There is no `app` folder in the default CRA template. Should I make one?

Reply:
Yeah, I want to do the same for the JS template.

If we were splitting out the root reducer separately, I'd put it in `app/rootReducer.js`.  I'd also tend towards putting `App.jsx` in that folder as well in a real project, but not sure it's necessary for the template.